solidity/deploy/05_deploy_bridge.ts

```diff
@@ -24,7 +24,11 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
     log: true,
   })
   const Redemption = await deploy("Redemption", { from: deployer, log: true })
-  const Wallets = await deploy("Wallets", { from: deployer, log: true })
+  const Wallets = await deploy("Wallets", {
+    contract: "contracts/bridge/Wallets.sol:Wallets",
```

Comment:
Why do we need it?

Reply:
>Wallets contract name duplicates with @keep-network/ecdsa. We need to be
specific in the deployment script which Wallets we want to deploy.

20ab23db71d5042164e047eab0eca1d6f49c7a75